sdk/ml/azure-ai-ml/.pre-commit-config.yaml

```diff
@@ -21,4 +21,54 @@ repos:
   hooks:
   - id: isort
     name: isort (python)
-    args: ['--settings-file', 'sdk/ml/azure-ai-ml/pyproject.toml']
\ No newline at end of file
+    args: ['--settings-file', 'sdk/ml/azure-ai-ml/pyproject.toml']
+- repo: local
+  hooks:
+    - id: pylint
+      name: pylint
+      entry: python
+      language: system
```

Comment:
just curious, can this not be done through azure SDK's tox commands?

Reply:
It can, but at the moment you'd be waiting about 5 minutes per commit. This because running `tox -e pylint` at the moment will:

* Unconditionally reinstall (almost) all dependencies
* Run against the entire SDK

I linked some issues in the description of this PR that describe these problems in a bit more detail.